src/main/java/duke/Launcher.java

```diff
@@ -0,0 +1,13 @@
+package duke;
+
+import duke.Duke;
+import javafx.application.Application;
+
+/**
+ * A launcher class to workaround classpath issues.
+ */
+public class Launcher {
+    public static void main(String[] args) {
+        Application.launch(Main.class, args);
+    }
+}
```

Comment:
You may configure your formatter to automatically add a new line on save (to make the display on GitHub more consistent).